frame/support/src/dispatch.rs

```diff
@@ -894,6 +901,7 @@ impl<T> PaysFee<T> for (u64, Pays) {
 /// * `integrity_test`: Executes in a test generated by `construct_runtime`, note it doesn't execute
 ///   in an externalities-provided environment. Implement
 ///   [`IntegrityTest`](./trait.IntegrityTest.html) trait.
+#[deprecated = "Please use new pallet attribute macro `#[pallet]` to declare pallet"]
```

Comment:
`decl_module` also declares extrinsics, yeah? If so, then we should also mention the `#[pallet::call]` attribute here as well.

Reply:
yeah, exactly